test/support/page_objects/visualize_page.js

```diff
@@ -565,12 +565,12 @@ export default class VisualizePage {
     .then(function getRect() {
       return self
       .setFindTimeout(defaultFindTimeout)
-      .findByCssSelector('rect.background')
+      .findByCssSelector('div.chart-wrapper > div.chart > svg')
```

Comment:
Not sure what this change is for? This method doesn't appear to be used in the scripted field tests.